org.srb2.SRB2Kart.metainfo.xml

```diff
@@ -32,9 +28,11 @@
   </provides>
   <releases>
     <release version="1.2" date="2020-05-12" />
+    <release version="1.1" date="2019-05-28" />
```

Comment:
We don't need past releases atm.

Reply:
I can't find it now, but I thought I remembered a blog post (possibly by @hughsie) claiming that Gnome Software does/would give some priority based on whether app had "momentum" (recent releases).  Anyway, its no big deal, I'll revert it.